src/libraries/System.Private.CoreLib/src/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -9,7 +10,7 @@ namespace System.Reflection
 {
     [Serializable]
     [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    public sealed class ReflectionTypeLoadException : SystemException, ISerializable
```

Comment:
I assume this is because Exception itself implements ISerializable?

Reply:
Yes. I asked @GrabYourPitchforks about the changes of this ilk yesterday and he said that if the base type already implemented `ISerializable` and the derived type was not altering the behavior, he removed the interface implementation from the derived type. Here and in a number of other places.